src/Networking/tests.cpp

```diff
@@ -142,16 +142,18 @@ TEST_CASE("Incremental PID", "[BaseStation]")
     p = popCANPacket();
     REQUIRE(GetDeviceSerialNumber(&p) == 1);
     REQUIRE(PacketIsOfID(&p, ID_MOTOR_UNIT_PID_POS_TGT_SET));
-    REQUIRE(GetPIDTargetFromPacket(&p) == start_angle + expected_increment);
+	REQUIRE(GetPIDTargetFromPacket(&p) ==
+			static_cast<unsigned>(start_angle + expected_increment));
```

Comment:
PID targets can be negative, so I don't think this is a good idea.

Actually wait, the HindsightCAN library says they are unsigned. Yikes, that seems like a bug in the library. I'll ask electronics about this.

Reply:
Okay I made a PR to fix this bug in the HindsightCAN repo: https://github.com/huskyroboticsteam/HindsightCAN/pull/32

Reply:
Sounds good. That kind of didn't make sense to me as an unsigned value so it's good that we caught that.